src/gtfs/read.rs

```diff
@@ -467,8 +467,10 @@ where
                 .index_mut(vj_idx)
                 .stop_times
                 .push(objects::StopTime {
```

Comment:
If we were to do that, I would implement the `Default` trait. That might actually be a good idea to implement it only with `#[cfg(test)]`.  What do you think @datanel @ArnaudOggy @patochectp ?

Reply:
am for to implement it only with test

Reply:
OK, I cannot implement `Default` because there is no default value possible for `stop_point_idx` (note that this would be the same problem with a `new()` or `minimal_new()` function).

As an alternative, I also thought about implementing `From<Idx<StopPoint>>` instead. But that isn't a solution either because most of the time, we do not set only the `stop_point_idx`.  This would lead to things like the following.

```rust
let mut stop_time = StopTime::from(stop_idx);
stop_time.departure_time = Time::new(0, 0, 0);
stop_time.arrival_time = Time::new(23, 59, 0);
```

I believe the ideal solution to this problem would be to have the [Builder Pattern](https://en.wikipedia.org/wiki/Builder_pattern) but there is much more coding involved (and since `Idx<StopPoint>` does not implement `Default`, it seems that automatic derive-crates like [derive_builder](https://crates.io/crates/derive_builder) and [typed_builder](https://crates.io/crates/typed-builder) won't handle the case).

```rust
let stop_time = StopTime::builder(stop_idx)
	.departure_time(Time::new(0, 0, 0))
	.arrival_time(Time::new(23, 59, 0))
	.build();
```

I think the best here is to keep like it is (but I'd be happy to code the builder pattern solution if you think it's worth it).

Reply:
OK, indeed I was thinking of a `.minimal_new(mandatory_param1, etc.)`.
Builder pattern looks way better, but I'd also keep it as-is: not sure the effort is worth the reward, and if we switch to an api/manager later it might be more painful to refactor it.